specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/storage.json

```diff
@@ -1375,8 +1448,14 @@
         },
         "isHnsEnabled": {
           "type": "boolean",
-           "x-ms-client-name": "IsHnsEnabled",
-           "description": "Account HierarchicalNamespace enabled if sets to true."
+          "x-ms-client-name": "IsHnsEnabled",
+          "description": "Account HierarchicalNamespace enabled if sets to true."
+        },
+        "failoverInProgress": {
```

Comment:
can you make this a generic property like "status" with states like failing over etc. It will allow you reuse the same property for other status related information in future.

Reply:
This status is a very special status just for Get LST from secondary location status. I don't see it will be reuse by any other APIs in the recent future. So I would like to keep it as what it is now.

Reply:
Ok lets atleast make this a string enum. bools are less descriptive and dont allow for future expansion of states.

Reply:
@ravbhatnagar 
Do you mean make the "failoverInProgress" a enum. Actually for this property's meaning, it almost won't have future expansion (won't add more value). It means if the failover is in progress. The value can only be "True" (means failover in progress) or null (means failover not in progress). I think a bool? can make user more easier to us the value in SDK, they only need to check the value is True, and don't need to compare the string is "True".

Reply:
Ok, thats fine. i just wanted to note that we are strongly recommending teams against using bools. Even if its just two states, string enums work much better. Its fine if you dont want to change it.

Reply:
Thanks！ I will take care of every bool value usage in swagger.